### ISSUE TITLE
When printing Reports, lots of white space present and the pages are only half-filled

### ISSUE THREAD

**Issue by marthakeezy** [msupply-foundation/open-msupply#5036]
## What went wrong? 😲

When printing `Reports`, lots of white space present and the pages are only half-filled.

## Expected behaviour 🤔

Can't think of a scenario when users need 50% of empty space, so suspect it's a bug-ish. Should we just have pages filled out?

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to `Reports -> Stock Detail`
2. In `Item Code or Name` type in something that will bring a lot of results, for example just a letter ('a')
3. See the report -> press `Print`
4. See the pages' layout - doesn't seem right

## Your environment 🌱

- Open mSupply Version: `V2.3.0-RC6`
- Legacy mSupply Central Server Version: `V7.17.10`

- Platform: Tablet (apk)

- Database type: SQLite3

**Comment by lache-melvin** [comment]
On tablet, you can change print orientation from portrait to landscape, and then it is fine. However, I'm not sure why it doesn't just include more rows on the portrait mode, whether that's something we've done in code?

**Comment by roxy-dao** [comment]
Pages are only filled with what you have filtered out.. the white space at the bottom of the page is because you don't have anymore lines... Would say this is expected

![Image](https://github.com/user-attachments/assets/920a11ea-c2b9-4e3d-be79-ca6143bef12f)

**Comment by marthakeezy** [comment]
Nah, it happens even if you have many items - just make sure you check on apk

I think the problem is that it defaults to portrait printing orientation - is there a way to set to landscape by default? But some clients might want to print in portrait, in which case the pages are only half-fileld

![Image](https://github.com/user-attachments/assets/2be240c6-23da-4771-aef1-518e1d6c612c)
